auth.css

```
@@ -37,17 +37,26 @@
 	background-color: var(--gray100);
 	padding: 16px 24px;
 	margin: 16px 0 24px;
+	border-radius: 12px;
 }
 
-.visible_on,
-.visible_off {
+.visible_button {
 	width: 24px;
 	height: 24px;
 	position: absolute;
 	z-index: 1;
 	top: 50%;
 	right: 24px;
 	margin-top: -16px;
+	padding: 0;
+	border: none;
+	cursor: pointer;
+}
+
+.visible_on,
+.visible_off {
+	width: 100%;
+	height: 100%;
```

💬 여담
해당 이미지의 경우 부모인 버튼요소에서 크기를 적절하게 제안해줘셔 이렇게 css를 작성하셔도 상관이 없지만
일반적으로 width, height 속성을 100%로 설정하면 기존 이미지의 비율이 깨질 수도 잇어 추천드리지 않습니다~
이러한 경우 width: 100% 만 있어도 height는 기존 이미지의 비율인 1:1을 유지하기 위해 자동으로 커집니다~

```suggestion
	width: 100%;
```